ethereum/contracts/relayer/coreRelayer/CoreRelayerMessages.sol

```diff
@@ -514,6 +527,34 @@ abstract contract CoreRelayerMessages is CoreRelayerGetters {
             ins.executionParameters.gasLimit);
     }
 
+    function decodeRedeliveryInstruction(bytes memory encoded) public pure returns (IWormholeRelayerInternalStructs.RedeliveryInstruction memory output) {
```

Comment:
Not actionable: 
It's interesting that not all decoders need to actually be part of the impl contract. if we run out of contract size again, we could consider moving these to the ForwardWrapper aka bucket-of-less-critical-code or consider moving them out of the deployed contract entirely.